src/middlewared/middlewared/plugins/vm/devices/pci.py

```diff
@@ -37,6 +37,10 @@ def reattach_device(self):
 
     def pre_start_vm_device_setup(self, *args, **kwargs):
         device = self.get_details()
+
+        if not self.not_in_use_by_vm():
```

Comment:
This is a double-negative which makes it pretty hard to follow what's actually being checked. I also think the name `not_in_use_by_vm()` is actually the inverse of what it's checking? It's actually checking if the device is IN USE by a vm? Either way, this is very confusing so we should rename/fix it.